entity-framework/core/modeling/backing-field.md

```diff
@@ -14,64 +14,53 @@ uid: core/modeling/backing-field
 > [!NOTE]
 > This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../ef6/index.md).
 
-When a backing field is configured, EF will write directly to that field when materializing entity instances from the database (rather than using the property setter). This is useful when there is no property setter, or the setter contains logic that should not be executed when setting initial property values for existing entities being loaded from the database.
+Backing fields allow EF to read and/or write to a field rather than a property.
 
-> [!WARNING]
-> The `ChangeTracker` has not yet been enabled to use backing fields when it needs to set the value of a property. This is only an issue for foreign key properties and generated properties - as the change tracker needs to propagate values into these properties. For these properties, a property setter must still be exposed.[Issue #4461](https://github.com/aspnet/EntityFramework/issues/4461) is tracking enabling the `ChangeTracker` to write to backing fields for properties with no setter.
+> [!NOTE]
+> Backing Field support was introduced in EF Core 1.1.0. If you are using an earlier release, the functionality shown in this article will not be available.
 
 ## Conventions
 
-**By convention, the following fields will be discovered as backing fields for a given property (listed in precedence order):**
-
-* <propertyName> differing only by case
```

Comment:
Somewhat confusing in the code, but actually all these matches are case sensitive. The property name match is not relevant except in weird cases, so it can probably omitted. The other matches are _ + camel-cased name, _ + name, m_ + camel-cased name, m_ + name.